unittesting/core/st3/fast_runner.py

```diff
@@ -0,0 +1,167 @@
+from functools import partial
+import time
+from unittest.runner import TextTestRunner, registerResult
+import warnings
+import sublime
+
+
+def defer(delay, callback, *args, **kwargs):
+    # Rely on late binding in case a user patches it
+    sublime.set_timeout(partial(callback, *args, **kwargs), delay)
+
+
+DEFAULT_CONDITION_POLL_TIME = 17
+DEFAULT_CONDITION_TIMEOUT = 4000
+AWAIT_WORKER = 'AWAIT_WORKER'
+# Extract `set_timeout_async`, t.i. *avoid* late binding, in case a user
+# patches it
+run_on_worker = sublime.set_timeout_async
+
+
+class FastDeferringTextTestRunner(TextTestRunner):
```

Comment:
How does it different from `DeferringTextTestRunner` except `DEFAULT_CONDITION_POLL_TIME`?
It seems that they are almost identical.